.gitignore

```diff
@@ -29,3 +29,5 @@ meetup_bot-*.tar
 *.db
 *.db-*
 *.backup.gz
+
+.elixir_ls/*
```

Comment:
This are not files that need to be ignored cause of the app itself, they are developer tooling or editor choices.

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

From https://git-scm.com/docs/gitignore

Reply:
TIL thank you!